File: Python Word examples/field/updateField.py
import spirecloudword
from spirecloudword.configuration import Configuration as WordConfiguration

appId = "your id"
appKey = "your key"

configuration = WordConfiguration(appId, appKey)
api = spirecloudword.api.fields_api.FieldsApi(configuration)

file_name = "Fields.docx"
extra = {'folder': 'input', 'dest_file_path': 'output/UpdateField.docx'}
result = api.update_field(file_name, **extra)

File: PHP Word Examples/FieldsApiDemo/updateField.php
<?php

use Spire\Cloud\Word\Sdk\Client\Api\FieldsApi;
use Spire\Cloud\Word\Sdk\Client\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl="https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey,$baseUrl);
$apiInstance = new FieldsApi($configuration);

$name = "UpdateField.docx";
$folder = "input";
$storage = null;
$password = null;
$dest_file_path = "output/UpdateField.docx";
$apiInstance->updateField($name, $dest_file_path, $folder, $storage, $password);